web-app/js/portal/prototypes/OpenLayers.js

```diff
@@ -119,6 +119,7 @@ OpenLayers.Layer.WMS.buildGetFeatureRequestUrl = function(baseUrl, layerName, ou
     wfsUrl += '&outputFormat=' + outputFormat;
     wfsUrl += '&REQUEST=GetFeature';
     wfsUrl += '&VERSION=1.0.0';
+    wfsUrl += '&maxFeatures=50';
```

Comment:
Better not to specify it here at all.  We are configuring this as a global setting in the server configuration.